standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/metrics/MetricsConstants.java

```diff
@@ -33,6 +33,10 @@
   public static final String OLDEST_ABORTED_TXN_ID = "oldest_aborted_txn_id";
   public static final String OLDEST_ABORTED_TXN_AGE = "oldest_aborted_txn_age_in_sec";
 
+  public static final String COMPACTION_NUM_OBSOLETE_DELTAS = COMPACTION_STATUS_PREFIX + "obsolete_deltas";
+  public static final String COMPACTION_NUM_DELTAS = COMPACTION_STATUS_PREFIX + "deltas";
```

Comment:
suggestion:  active_deltas instead of deltas
is more clear

Reply:
fixed